portal/portal-sprint-98.md

```diff
@@ -0,0 +1,24 @@
+The goal this past Sprint was to work on the migration of the Queue page to React, as part of our effort to improve performance of roll-out relevant Portal sections.
+
+## User Management
+
+### Email confirmation
+
+Very often customers were entering accidentally a misspelled email address and were thus unable to login. By adding a confirmation email so the user has to submit it for a second time. An error will be issued if the two entries don’t match 
+
+![VerificationEmail](images/sprint-98/VerificationEmail.png)
+
+### New React-based index page for Queue DE
```

Comment:
We should focus on the performance improvements that we did. I don't think that customers do care about the framework we are using :)

Reply:
Ok, I modified it a bit and I think now it reflects more what you are saying! Do you think it is ok now?